backend/layers/business/business.py

```diff
@@ -242,6 +245,11 @@ def update_collection_version(self, version_id: CollectionVersionId, body: Colle
                     value.strip_fields()
                 setattr(new_metadata, field, value)
 
+        # Issue all updates
+        if unset_publisher_metadata:
+            self.database_provider.save_collection_publisher_metadata(version_id, None)
+        if set_publisher_metadata is not None:
```

Comment:
nit: i believe its not possible for both to be true as written, but its easier to quickly understand the intention if we make this an elif